Description: Do not pull revision info from GIT
 Stop trying to pull version info fom GIT  when autoconf is run.
Author: Andreas Metzler <ametzler@debian.org>
Origin: vendor
Forwarded: no
Last-Update: 2016-02-21

--- a/configure.ac
+++ b/configure.ac
@@ -39,12 +39,14 @@ m4_define(mym4_version_micro, [0])
 m4_define(mym4_version,
           [mym4_version_major.mym4_version_minor.mym4_version_micro])
 m4_define([mym4_revision],
-          m4_esyscmd([git rev-parse --short HEAD | tr -d '\n\r']))
+	  [1092])
+dnl	  m4_esyscmd([git rev-parse --short HEAD | tr -d '\n\r']))
 m4_define([mym4_revision_dec],
           m4_esyscmd_s([echo $((0x$(echo ]mym4_revision[|head -c 4)))]))
 m4_define([mym4_betastring],
-          m4_esyscmd_s([git describe --match 'libgcrypt-[0-9].*[0-9]' --long|\
-                        awk -F- '$3!=0{print"-beta"$3}']))
+	  [-beta])
+dnl          m4_esyscmd_s([git describe --match 'libgcrypt-[0-9].*[0-9]' --long|\
+dnl                        awk -F- '$3!=0{print"-beta"$3}']))
 m4_define([mym4_isgit],m4_if(mym4_betastring,[],[no],[yes]))
 m4_define([mym4_full_version],[mym4_version[]mym4_betastring])
 
